### ISSUE TITLE
atom-beautify incorrectly beautifies `alias` in ruby, contradicting rubocop

### ISSUE THREAD

**Issue by moskyb** [Glavin001/atom-beautify#1456]
# Description

The results of beautification are not what I expect.

# Input Before Beautification

This is what the code looked like before:

```
  alias invite_to_join_by! invite!
```

# Expected Output

The beautified code should have looked like this:

```
  alias invite_to_join_by! invite!
```

# Actual Output

The beautified code actually looked like this:

```
  alias_method :invite_to_join_by!, :invite!
```

# Steps to Reproduce

1. Add code to Atom editor
2. Run command `Atom Beautify: Beautify Editor`
3. This beautified code does not look right!

# Debug

Here is a link to the `debug.md` Gist: https://gist.github.com/moskyb/449ce5370c4e9ca1e665c48f122718bd

# Checklist

I have:
- [x] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [x] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [x] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [x] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [x] Generated debugging information and added link for `debug.md` Gist to this issue

**Comment by Glavin001** [comment]
From your debugging logs:

```
2017-01-10T01:01:54.557Z - debug: [beautifiers/beautifier.coffee] spawn /Users/bmosky/.rbenv/shims/rubocop 0=--auto-correct, 1=--config, 2=/Users/bmosky/Carnival/carnival-consumers/.rubocop.yml, 3=/tmp/temp117010-860-wj05sx.hetzgjh5mi
2017-01-10T01:01:57.321Z - debug: [beautifiers/beautifier.coffee] spawn done 1 warning: parser/current is loading parser/ruby22, which recognizes
warning: 2.2-compliant syntax, but you are running 2.3.2.
 Inspecting 1 file
C

Offenses:

mp/temp117010-860-wj05sx.hetzgjh5mi:1:1: C: Use snake_case for source file names.
require_dependency 'concerns/devise_mailable'
^
mp/temp117010-860-wj05sx.hetzgjh5mi:104:3: C: [Corrected] Use alias_method instead of alias.
  alias invite_to_join_by! invite!
  ^^^^^

1 file inspected, 2 offenses detected, 1 offense corrected

2017-01-10T01:01:57.321Z - verbose: [beautifiers/beautifier.coffee] spawn result 1 Inspecting 1 file
C

Offenses:

mp/temp117010-860-wj05sx.hetzgjh5mi:1:1: C: Use snake_case for source file names.
require_dependency 'concerns/devise_mailable'
^
mp/temp117010-860-wj05sx.hetzgjh5mi:104:3: C: [Corrected] Use alias_method instead of alias.
  alias invite_to_join_by! invite!
  ^^^^^

1 file inspected, 2 offenses detected, 1 offense corrected
 warning: parser/current is loading parser/ruby22, which recognizes
warning: 2.2-compliant syntax, but you are running 2.3.2.
```

It looks like Atom-Beautify worked.

If it is an issue with the options Atom-Beautify passes to Rubocop, then https://github.com/Glavin001/atom-beautify/pull/1776 may resolve your issue.

Otherwise, I recommend making an issue over at https://github.com/bbatsov/rubocop

**Comment by Glavin001** [comment]
See #1776 

Published to v0.32.0
